### PR TITLE
[EDCR] Add new object type for EDCR

### PR DESCRIPTION
**Pull request by KUMARMUKULSAP**

### CODE REVIEW

**Comment by GuilhermeSaraiva96** [comment]
Hi,

Maybe the Pull Request is not yet review-ready, but the AFF contains no information, except for the type name. Is this on purpose?

**Review by GuilhermeSaraiva96 — CHANGES_REQUESTED** [review]
Requesting changes so that my last question is clarified

**Comment by KUMARMUKULSAP** [comment]
There are no additional attributes to this object, we only need description of the object

File: file-formats/edcr/type/zif_aff_edcr_v1.intf.abap [hunk 1]
```diff
@@ -0,0 +1,16 @@
+INTERFACE zif_aff_edcr_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Representation Type</p>
+    "! Representation Type (EDCR) v1
```
Comment (Markus1812) [inline file-formats/edcr/type/zif_aff_edcr_v1.intf.abap:6]:
```suggestion
    "! Representation type
```
Please use sentence case in descriptions as described in our [wiki](https://github.com/SAP/abap-file-formats/blob/main/docs/json.md#description).

**Review by Markus1812 — CHANGES_REQUESTED** [review]
Thank you for your AFF type. Please delete the .DS_Store file and add an example with the name `z_aff_example_edcr.edcr.json`, even though its nearly empty. Remember to link it in the README.md.

**Comment by KUMARMUKULSAP** [comment]
> Thank you for your AFF type. Please delete the .DS_Store file and add an example with the name `z_aff_example_edcr.edcr.json`, even though its nearly empty. Remember to link it in the README.md.

**Comment by Markus1812** [comment]
Thank you for adapting our suggestions. We have fixed all line endings of the example and README file and added a new line at the end of file. 

All checks except Contributor License Agreement are now fulfilled. If you could please check the [first comment](https://github.com/SAP/abap-file-formats/pull/630#issuecomment-2210175758) and perform the necessary actions.

**Comment by KUMARMUKULSAP** [comment]
@Markus1812 : I have signed the CLA agreement.

**Review by wurzka — APPROVED** [review]
Looks good to me.

**Review by GuilhermeSaraiva96 — APPROVED** [review]

**Review by wurzka — APPROVED** [review]

**Pull request merged**